### PR TITLE
Add option to overwrite existing content in importCSAR dialog

### PR DESCRIPTION
**Pull request by TStadelmaier**
Signed-off-by: Tino Stadelmaier <tstadelmaier.github@gmail.com>

screenshot:
supportoverwrite

### CODE REVIEW

**Comment by genie-winery** [comment]
Can one of the admins verify this patch?

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ This is similar to [Angular's CHANGELOG.md](https://github.com/angular/angular/b
 ## [unreleased]
 
 ### Changed
+- Add overwrite option in importCSAR dialog
```
Comment (koppor) [inline CHANGELOG.md]: "dialog for importing CSARs"

File: org.eclipse.winery.repository.rest/src/test/java/org/eclipse/winery/repository/rest/resources/AbstractResourceTest.java [hunk 2]
```diff
@@ -241,6 +241,15 @@ protected void assertPost(String restUrl, Path file) {
 			.statusCode(201);
 	}
 
+	protected void assertPostWithOverwrite(String restUrl, Path file, boolean overwrite) {
```
Comment (koppor) [inline org.eclipse.winery.repository.rest/src/test/java/org/eclipse/winery/repository/rest/resources/AbstractResourceTest.java]: Move to test class; if possible. This thing is required only once. you can make `start()` protected.

File: org.eclipse.winery.repository.rest/src/test/java/org/eclipse/winery/repository/rest/resources/MainResourceTest.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import org.eclipse.jetty.toolchain.test.MavenTestingUtils;
+import org.junit.Test;
+
+public class MainResourceTest extends AbstractResourceTest {
+	@Test
+	public void importCSARTestWithOverwrite() throws Exception {
+		setRevisionTo("dc30db8f6086a8bcf6b39881d124f15fb05168f4");
+		this.assertGet("servicetemplates/http%253A%252F%252Fplain.winery.opentosca.org%252Fservicetemplates/ImportCsarWithOverwriteTest/", "entitytypes/servicetemplates/importCsarWithOverwriteTest_initial.json");
+		final Path path = MavenTestingUtils.getProjectFilePath("src/test/resources/entitytypes/servicetemplates/ImportCsarOverwriteTest.csar");
+		this.assertPostWithOverwrite("",path,true);
```
Comment (koppor) [inline org.eclipse.winery.repository.rest/src/test/java/org/eclipse/winery/repository/rest/resources/MainResourceTest.java]: ctrl+alt+l to ensure proper spacing

File: org.eclipse.winery.repository.rest/src/test/java/org/eclipse/winery/repository/rest/resources/AbstractResourceTest.java (hunk 2 above)
Comment (koppor) [reply org.eclipse.winery.repository.rest/src/test/java/org/eclipse/winery/repository/rest/resources/AbstractResourceTest.java]: Please really remove the test method here 😇

**Comment by koppor** [comment]
Minor comment not addressed yet, therefore not merged.

File: org.eclipse.winery.repository.rest/src/test/java/org/eclipse/winery/repository/rest/resources/AbstractResourceTest.java (hunk 2 above)
Comment (TStadelmaier) [reply org.eclipse.winery.repository.rest/src/test/java/org/eclipse/winery/repository/rest/resources/AbstractResourceTest.java]: Sorry, forgot to do this. Will be done in a few minutes.

**Pull request merged**